hardhat/tasks/sips/args/sipArgs.js

```diff
@@ -1104,6 +1104,73 @@ const getArgsSIP0077 = async (hre) => {
     return { args, governor: "GovernorAdmin" };
 };
 
+const getArgsSip0084 = async (hre) => {
+    const {
+        ethers,
+        deployments: { get, log },
+    } = hre;
+
+    const abiCoder = new ethers.utils.AbiCoder();
+    const priceFeeds = await ethers.getContract("PriceFeeds");
+    const priceFeedsOwner = await priceFeeds.owner();
+
+    const wrbtcAddress = (await get("WRBTC")).address;
+    const priceFeedMocAddress = (await get("PriceFeedsMoC")).address;
+
+    const currentWrbtcPriceFeed = await priceFeeds.pricesFeeds(wrbtcAddress);
+    if (currentWrbtcPriceFeed.toLowerCase() === priceFeedMocAddress.toLowerCase()) {
+        throw new Error(
+            `new wrbtc priceFeed ${priceFeedMocAddress} could not be the same with the current one ${currentWrbtcPriceFeed}`
+        );
+    }
+
+    const args = {
+        targets: [priceFeeds.address],
+        targetOwnerValidationAddresses: [priceFeedsOwner],
+        values: [0],
+        signatures: ["setPriceFeed(address[],address[])"],
+        data: [
+            abiCoder.encode(["address[]", "address[]"], [[wrbtcAddress], [priceFeedMocAddress]]),
+        ],
+        description:
+            // @todo update description
+            "SIP-0084: Update MoCPriceFeeds, Details: https://github.com/DistributedCollective/SIPS/blob/8cb4f72/SIP-0084.md, sha256: ",
+    };
+    return { args, governor: "GovernorAdmin" };
+};
+
+const getArgsSip0085 = async (hre) => {
+    const {
+        ethers,
+        deployments: { get },
+    } = hre;
+    const abiCoder = new ethers.utils.AbiCoder();
+
+    const zeroPriceFeed = await ethers.getContract("ZeroPriceFeed");
+    const zeroPriceFeedImplementation = await ethers.getContract("ZeroPriceFeed_Implementation"); // @todo update the address in the deployment file
+
```

Comment:
pls add check that the implementation is different from the active implementation being replaced